sdcm/sct_events/database.py

```diff
@@ -381,6 +381,12 @@ def __init__(self, node: str, severity=Severity.NORMAL, **__):
         super().__init__(node=node, severity=severity, publish_event=True)
 
 
+class ScyllaYamlUpdateEvent(ScyllaDatabaseContinuousEvent):
+    def __init__(self, node: str, message: Optional[str] = None, severity=Severity.NORMAL, **__):
+        super().__init__(node=node, severity=severity)
+        self.message = message or f"Updating scylla.yaml contents on node: {node}"
```

Comment:
I'm guessing if we could also add the actual change it might be a big more useful

Reply:
Agreed, adding the content of the updates to the message.